tests/pipelines/test_pipelines.py

```diff
@@ -821,7 +821,12 @@ def test_custom_model_and_pipeline(self):
             pipe_new = CustomPipeline.from_pretrained(tmpdirname)
             pipe_new.save_pretrained(tmpdirname)
 
-        assert dict(pipe_new.config) == dict(pipe.config)
+        conf_1 = dict(pipe.config)
+        conf_2 = dict(pipe_new.config)
+
+        del conf_2["_name_or_path"]
+
+        assert conf_1 == conf_2
```

Comment:
Gives us a good opportunity to brainstorm `DiffusionPipeline.from_config(pipe.config)`.

Reply:
Don't fully understand here